docs/scaleway/bucket_policy_production.json

```diff
@@ -12,6 +12,26 @@
          "Resource":[
             "transport-prod-backups/*"
          ]
+      },
+      {
+         "Sid":"Get access to all objects",
+         "Effect":"Allow",
+         "Principal":{
+            "SCW":"project_id:37d278a7-e548-4029-a58d-111bdcf23d46"
+         },
+         "Action":"s3:GetObject",
+         "Resource":[
+            "transport-prod-backups/*"
+         ]
+      },
+      {
+         "Sid":"Objects cannot be deleted",
+         "Effect":"Deny",
+         "Principal":"*",
+         "Action":"s3:DeleteObject",
+         "Resource":[
+            "transport-prod-backups/*"
+         ]
```

Comment:
J'ai demandé la confirmation que les lifecyle rules s'appliquent bien même avec ça.

---
[Voir ticket](https://console.scaleway.com/support/tickets/BJIQ-0297-FJSC)

Hello,

I've got a bucket policy with this statement.

```
{
   "Sid":"Objects cannot be deleted",
   "Effect":"Deny",
   "Principal":"*",
   "Action":"s3:DeleteObject",
   "Resource":[
      "transport-prod-backups/*"
   ]
}
```

Can you confirm that if I have a lifecycle rule to delete objects after a number of days, the lifecycle rule will indeed be applied and delete objects?

It seems to be the case on AWS S3 https://stackoverflow.com/questions/67092064/does-s3-lifecycle-rules-overwrite-deny-delete-bucket-or-deleteobject-policy-is-s but I want to make sure it works as well here.